src/create-model.jl

```diff
@@ -662,10 +671,20 @@ function create_model(
                         base_name = "flow[($(row.from), $(row.to)), $(row.year), $(row.rep_period), $(row.timesteps_block)]"
                     ) for row in eachrow(df_flows)
                 ]
-        @variable(model, 0 ≤ assets_investment[y in Y, a in Ai[y]])  #number of installed asset units [N]
         @variable(model, 0 ≤ flows_investment[y in Y, (u, v) in Fi[y]])
+
+        ### Investment variables
+        @variable(model, 0 ≤ assets_investment[y in Y, a in Ai[y]])  #number of installed asset units [N]
+        @variable(
+            model,
+            0 ≤ assets_decommission_simple_method[
+                y in Y,
+                a in investable_assets_using_simple_method[y],
```

Comment:
I think we need a different set here for a. We want to be able to decommission both investable assets and existing assets. Suppose we have some gas units that we don't want to invest in, but they are still there an can be decommissioned. That's why in the other PR, I made a comment/to-do, to define a set for A that covers both investable assets and existing assets.

Reply:
Latest changes account for the reviewed formulation